docs/installation-guide-mac.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
+
+Installation guide for macOS
+============================
+
+Follow these instructions to install the CityEnergyAnalyst (CEA) on a macOS system (tested with Mojave 10.14.5) from the source
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Download and install Git (64-bit).
+* Download and install Github Desktop (64-bit).
+* Download and install Miniconda(64-bit) for Python 2.7.

```

Comment:
the .pkg-installer is the easiest

Reply:
Not sure what that means.

Reply:
there were two installers for miniconda. a bash installer and a .pkg installer. the bash installer needs to be installed from the terminal. which one did you use? (following my Docker example, probably the bash one? with curl to get it?